<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<!-- NewPage -->
<html lang="en">
<head>
<!-- Generated by javadoc (version 1.7.0_17) on Mon Feb 17 10:36:01 CET 2014 -->
<title>com.ohnosequences.bio4j.neo4j.model.relationships.features</title>
<meta name="date" content="2014-02-17">
<link rel="stylesheet" type="text/css" href="../../../../../../../stylesheet.css" title="Style">
</head>
<body>
<script type="text/javascript"><!--
    if (location.href.indexOf('is-external=true') == -1) {
        parent.document.title="com.ohnosequences.bio4j.neo4j.model.relationships.features";
    }
//-->
</script>
<noscript>
<div>JavaScript is disabled on your browser.</div>
</noscript>
<!-- ========= START OF TOP NAVBAR ======= -->
<div class="topNav"><a name="navbar_top">
<!--   -->
</a><a href="#skip-navbar_top" title="Skip navigation links"></a><a name="navbar_top_firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="../../../../../../../overview-summary.html">Overview</a></li>
<li class="navBarCell1Rev">Package</li>
<li>Class</li>
<li><a href="package-use.html">Use</a></li>
<li><a href="package-tree.html">Tree</a></li>
<li><a href="../../../../../../../deprecated-list.html">Deprecated</a></li>
<li><a href="../../../../../../../index-files/index-1.html">Index</a></li>
<li><a href="../../../../../../../help-doc.html">Help</a></li>
</ul>
</div>
<div class="subNav">
<ul class="navList">
<li><a href="../../../../../../../com/ohnosequences/bio4j/neo4j/model/relationships/comment/package-summary.html">Prev Package</a></li>
<li><a href="../../../../../../../com/ohnosequences/bio4j/neo4j/model/relationships/go/package-summary.html">Next Package</a></li>
</ul>
<ul class="navList">
<li><a href="../../../../../../../index.html?com/ohnosequences/bio4j/neo4j/model/relationships/features/package-summary.html" target="_top">Frames</a></li>
<li><a href="package-summary.html" target="_top">No Frames</a></li>
</ul>
<ul class="navList" id="allclasses_navbar_top">
<li><a href="../../../../../../../allclasses-noframe.html">All Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_top");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
</div>
<a name="skip-navbar_top">
<!--   -->
</a></div>
<!-- ========= END OF TOP NAVBAR ========= -->
<div class="header">
<h1 title="Package" class="title">Package&nbsp;com.ohnosequences.bio4j.neo4j.model.relationships.features</h1>
<div class="docSummary">
<div class="block">Protein features relationships</div>
</div>
<p>See:&nbsp;<a href="#package_description">Description</a></p>
</div>
<div class="contentContainer">
<ul class="blockList">
<li class="blockList">
<table class="packageSummary" border="0" cellpadding="3" cellspacing="0" summary="Class Summary table, listing classes, and an explanation">
<caption><span>Class Summary</span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Class</th>
<th class="colLast" scope="col">Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><a href="../../../../../../../com/ohnosequences/bio4j/neo4j/model/relationships/features/ActiveSiteFeatureRel.html" title="class in com.ohnosequences.bio4j.neo4j.model.relationships.features">ActiveSiteFeatureRel</a></td>
<td class="colLast">
<div class="block">Amino acid(s) directly involved in the activity of an enzyme</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../../../../../../com/ohnosequences/bio4j/neo4j/model/relationships/features/BasicFeatureRel.html" title="class in com.ohnosequences.bio4j.neo4j.model.relationships.features">BasicFeatureRel</a></td>
<td class="colLast">
<div class="block">Basic feature relationship</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../../../../../../com/ohnosequences/bio4j/neo4j/model/relationships/features/BindingSiteFeatureRel.html" title="class in com.ohnosequences.bio4j.neo4j.model.relationships.features">BindingSiteFeatureRel</a></td>
<td class="colLast">
<div class="block">Binding site for any chemical group (co-enzyme, prosthetic group, etc.)</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../../../../../../com/ohnosequences/bio4j/neo4j/model/relationships/features/CalciumBindingRegionFeatureRel.html" title="class in com.ohnosequences.bio4j.neo4j.model.relationships.features">CalciumBindingRegionFeatureRel</a></td>
<td class="colLast">
<div class="block">Position(s) of calcium binding region(s) within the protein</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../../../../../../com/ohnosequences/bio4j/neo4j/model/relationships/features/ChainFeatureRel.html" title="class in com.ohnosequences.bio4j.neo4j.model.relationships.features">ChainFeatureRel</a></td>
<td class="colLast">
<div class="block">Extent of a polypeptide chain in the mature protein</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../../../../../../com/ohnosequences/bio4j/neo4j/model/relationships/features/CoiledCoilRegionFeatureRel.html" title="class in com.ohnosequences.bio4j.neo4j.model.relationships.features">CoiledCoilRegionFeatureRel</a></td>
<td class="colLast">
<div class="block">Positions of regions of coiled coil within the protein</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../../../../../../com/ohnosequences/bio4j/neo4j/model/relationships/features/CompositionallyBiasedRegionFeatureRel.html" title="class in com.ohnosequences.bio4j.neo4j.model.relationships.features">CompositionallyBiasedRegionFeatureRel</a></td>
<td class="colLast">
<div class="block">Region of compositional bias in the protein</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../../../../../../com/ohnosequences/bio4j/neo4j/model/relationships/features/CrossLinkFeatureRel.html" title="class in com.ohnosequences.bio4j.neo4j.model.relationships.features">CrossLinkFeatureRel</a></td>
<td class="colLast">
<div class="block">Residues participating in covalent linkage(s) between proteins</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../../../../../../com/ohnosequences/bio4j/neo4j/model/relationships/features/DisulfideBondFeatureRel.html" title="class in com.ohnosequences.bio4j.neo4j.model.relationships.features">DisulfideBondFeatureRel</a></td>
<td class="colLast">
<div class="block">Cysteine residues participating in disulfide bonds</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../../../../../../com/ohnosequences/bio4j/neo4j/model/relationships/features/DnaBindingRegionFeatureRel.html" title="class in com.ohnosequences.bio4j.neo4j.model.relationships.features">DnaBindingRegionFeatureRel</a></td>
<td class="colLast">
<div class="block">Position and type of a DNA-binding domain</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../../../../../../com/ohnosequences/bio4j/neo4j/model/relationships/features/DomainFeatureRel.html" title="class in com.ohnosequences.bio4j.neo4j.model.relationships.features">DomainFeatureRel</a></td>
<td class="colLast">
<div class="block">Position and type of each modular protein domain</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../../../../../../com/ohnosequences/bio4j/neo4j/model/relationships/features/GlycosylationSiteFeatureRel.html" title="class in com.ohnosequences.bio4j.neo4j.model.relationships.features">GlycosylationSiteFeatureRel</a></td>
<td class="colLast">
<div class="block">Covalently attached glycan group(s)</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../../../../../../com/ohnosequences/bio4j/neo4j/model/relationships/features/HelixFeatureRel.html" title="class in com.ohnosequences.bio4j.neo4j.model.relationships.features">HelixFeatureRel</a></td>
<td class="colLast">
<div class="block">Helical regions within the experimentally determined protein structure</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../../../../../../com/ohnosequences/bio4j/neo4j/model/relationships/features/InitiatorMethionineFeatureRel.html" title="class in com.ohnosequences.bio4j.neo4j.model.relationships.features">InitiatorMethionineFeatureRel</a></td>
<td class="colLast">
<div class="block">Cleavage of the initiator methionine</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../../../../../../com/ohnosequences/bio4j/neo4j/model/relationships/features/IntramembraneRegionFeatureRel.html" title="class in com.ohnosequences.bio4j.neo4j.model.relationships.features">IntramembraneRegionFeatureRel</a></td>
<td class="colLast">
<div class="block">Extent of a region located in a membrane without crossing it</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../../../../../../com/ohnosequences/bio4j/neo4j/model/relationships/features/LipidMoietyBindingRegionFeatureRel.html" title="class in com.ohnosequences.bio4j.neo4j.model.relationships.features">LipidMoietyBindingRegionFeatureRel</a></td>
<td class="colLast">
<div class="block">Covalently attached lipid group(s)</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../../../../../../com/ohnosequences/bio4j/neo4j/model/relationships/features/MetalIonBindingSiteFeatureRel.html" title="class in com.ohnosequences.bio4j.neo4j.model.relationships.features">MetalIonBindingSiteFeatureRel</a></td>
<td class="colLast">
<div class="block">Binding site for a metal ion</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../../../../../../com/ohnosequences/bio4j/neo4j/model/relationships/features/ModifiedResidueFeatureRel.html" title="class in com.ohnosequences.bio4j.neo4j.model.relationships.features">ModifiedResidueFeatureRel</a></td>
<td class="colLast">
<div class="block">Modified residues excluding lipids, glycans and protein cross-links</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../../../../../../com/ohnosequences/bio4j/neo4j/model/relationships/features/MutagenesisSiteFeatureRel.html" title="class in com.ohnosequences.bio4j.neo4j.model.relationships.features">MutagenesisSiteFeatureRel</a></td>
<td class="colLast">
<div class="block">Site which has been experimentally altered by mutagenesis</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../../../../../../com/ohnosequences/bio4j/neo4j/model/relationships/features/NonConsecutiveResiduesFeatureRel.html" title="class in com.ohnosequences.bio4j.neo4j.model.relationships.features">NonConsecutiveResiduesFeatureRel</a></td>
<td class="colLast">
<div class="block">Indicates that two residues in a sequence are not consecutive</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../../../../../../com/ohnosequences/bio4j/neo4j/model/relationships/features/NonStandardAminoAcidFeatureRel.html" title="class in com.ohnosequences.bio4j.neo4j.model.relationships.features">NonStandardAminoAcidFeatureRel</a></td>
<td class="colLast">
<div class="block">Occurence of non-standard amino acids (selenocysteine and pyrrolysine) in the protein sequence</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../../../../../../com/ohnosequences/bio4j/neo4j/model/relationships/features/NonTerminalResidueFeatureRel.html" title="class in com.ohnosequences.bio4j.neo4j.model.relationships.features">NonTerminalResidueFeatureRel</a></td>
<td class="colLast">
<div class="block">The sequence is incomplete.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../../../../../../com/ohnosequences/bio4j/neo4j/model/relationships/features/NucleotidePhosphateBindingRegionFeatureRel.html" title="class in com.ohnosequences.bio4j.neo4j.model.relationships.features">NucleotidePhosphateBindingRegionFeatureRel</a></td>
<td class="colLast">
<div class="block">Nucleotide phosphate binding region</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../../../../../../com/ohnosequences/bio4j/neo4j/model/relationships/features/PeptideFeatureRel.html" title="class in com.ohnosequences.bio4j.neo4j.model.relationships.features">PeptideFeatureRel</a></td>
<td class="colLast">
<div class="block">Extent of an active peptide in the mature protein</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../../../../../../com/ohnosequences/bio4j/neo4j/model/relationships/features/PropeptideFeatureRel.html" title="class in com.ohnosequences.bio4j.neo4j.model.relationships.features">PropeptideFeatureRel</a></td>
<td class="colLast">
<div class="block">Part of a protein that is cleaved during maturation or activation</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../../../../../../com/ohnosequences/bio4j/neo4j/model/relationships/features/RegionOfInterestFeatureRel.html" title="class in com.ohnosequences.bio4j.neo4j.model.relationships.features">RegionOfInterestFeatureRel</a></td>
<td class="colLast">
<div class="block">Region of interest in the sequence</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../../../../../../com/ohnosequences/bio4j/neo4j/model/relationships/features/RepeatFeatureRel.html" title="class in com.ohnosequences.bio4j.neo4j.model.relationships.features">RepeatFeatureRel</a></td>
<td class="colLast">
<div class="block">Positions of repeated sequence motifs or repeated domains</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../../../../../../com/ohnosequences/bio4j/neo4j/model/relationships/features/SequenceConflictFeatureRel.html" title="class in com.ohnosequences.bio4j.neo4j.model.relationships.features">SequenceConflictFeatureRel</a></td>
<td class="colLast">
<div class="block">Description of sequence discrepancies of unknown origin</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../../../../../../com/ohnosequences/bio4j/neo4j/model/relationships/features/SequenceVariantFeatureRel.html" title="class in com.ohnosequences.bio4j.neo4j.model.relationships.features">SequenceVariantFeatureRel</a></td>
<td class="colLast">
<div class="block">Description of a natural variant of the protein</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../../../../../../com/ohnosequences/bio4j/neo4j/model/relationships/features/ShortSequenceMotifFeatureRel.html" title="class in com.ohnosequences.bio4j.neo4j.model.relationships.features">ShortSequenceMotifFeatureRel</a></td>
<td class="colLast">
<div class="block">Short (up to 20 amino acids) sequence motif of biological interest</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../../../../../../com/ohnosequences/bio4j/neo4j/model/relationships/features/SignalPeptideFeatureRel.html" title="class in com.ohnosequences.bio4j.neo4j.model.relationships.features">SignalPeptideFeatureRel</a></td>
<td class="colLast">
<div class="block">Sequence targeting proteins to the secretory pathway or periplasmic space</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../../../../../../com/ohnosequences/bio4j/neo4j/model/relationships/features/SiteFeatureRel.html" title="class in com.ohnosequences.bio4j.neo4j.model.relationships.features">SiteFeatureRel</a></td>
<td class="colLast">
<div class="block">Any interesting single amino acid site on the sequence</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../../../../../../com/ohnosequences/bio4j/neo4j/model/relationships/features/SpliceVariantFeatureRel.html" title="class in com.ohnosequences.bio4j.neo4j.model.relationships.features">SpliceVariantFeatureRel</a></td>
<td class="colLast">
<div class="block">Amino acid change(s) producing alternate protein isoforms</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../../../../../../com/ohnosequences/bio4j/neo4j/model/relationships/features/StrandFeatureRel.html" title="class in com.ohnosequences.bio4j.neo4j.model.relationships.features">StrandFeatureRel</a></td>
<td class="colLast">
<div class="block">Beta strand regions within the experimentally determined protein structure</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../../../../../../com/ohnosequences/bio4j/neo4j/model/relationships/features/TopologicalDomainFeatureRel.html" title="class in com.ohnosequences.bio4j.neo4j.model.relationships.features">TopologicalDomainFeatureRel</a></td>
<td class="colLast">
<div class="block">Location of non-membrane regions of membrane-spanning proteins</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../../../../../../com/ohnosequences/bio4j/neo4j/model/relationships/features/TransitPeptideFeatureRel.html" title="class in com.ohnosequences.bio4j.neo4j.model.relationships.features">TransitPeptideFeatureRel</a></td>
<td class="colLast">
<div class="block">Extent of a transit peptide for organelle targeting</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../../../../../../com/ohnosequences/bio4j/neo4j/model/relationships/features/TransmembraneRegionFeatureRel.html" title="class in com.ohnosequences.bio4j.neo4j.model.relationships.features">TransmembraneRegionFeatureRel</a></td>
<td class="colLast">
<div class="block">Extent of a membrane-spanning region</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../../../../../../com/ohnosequences/bio4j/neo4j/model/relationships/features/TurnFeatureRel.html" title="class in com.ohnosequences.bio4j.neo4j.model.relationships.features">TurnFeatureRel</a></td>
<td class="colLast">
<div class="block">Turns within the experimentally determined protein structure</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="../../../../../../../com/ohnosequences/bio4j/neo4j/model/relationships/features/UnsureResidueFeatureRel.html" title="class in com.ohnosequences.bio4j.neo4j.model.relationships.features">UnsureResidueFeatureRel</a></td>
<td class="colLast">
<div class="block">Regions of uncertainty in the sequence</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="../../../../../../../com/ohnosequences/bio4j/neo4j/model/relationships/features/ZincFingerRegionFeatureRel.html" title="class in com.ohnosequences.bio4j.neo4j.model.relationships.features">ZincFingerRegionFeatureRel</a></td>
<td class="colLast">
<div class="block">Position(s) and type(s) of zinc fingers within the protein</div>
</td>
</tr>
</tbody>
</table>
</li>
</ul>
<a name="package_description">
<!--   -->
</a>
<h2 title="Package com.ohnosequences.bio4j.neo4j.model.relationships.features Description">Package com.ohnosequences.bio4j.neo4j.model.relationships.features Description</h2>
<div class="block">Protein features relationships</div>
</div>
<!-- ======= START OF BOTTOM NAVBAR ====== -->
<div class="bottomNav"><a name="navbar_bottom">
<!--   -->
</a><a href="#skip-navbar_bottom" title="Skip navigation links"></a><a name="navbar_bottom_firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="../../../../../../../overview-summary.html">Overview</a></li>
<li class="navBarCell1Rev">Package</li>
<li>Class</li>
<li><a href="package-use.html">Use</a></li>
<li><a href="package-tree.html">Tree</a></li>
<li><a href="../../../../../../../deprecated-list.html">Deprecated</a></li>
<li><a href="../../../../../../../index-files/index-1.html">Index</a></li>
<li><a href="../../../../../../../help-doc.html">Help</a></li>
</ul>
</div>
<div class="subNav">
<ul class="navList">
<li><a href="../../../../../../../com/ohnosequences/bio4j/neo4j/model/relationships/comment/package-summary.html">Prev Package</a></li>
<li><a href="../../../../../../../com/ohnosequences/bio4j/neo4j/model/relationships/go/package-summary.html">Next Package</a></li>
</ul>
<ul class="navList">
<li><a href="../../../../../../../index.html?com/ohnosequences/bio4j/neo4j/model/relationships/features/package-summary.html" target="_top">Frames</a></li>
<li><a href="package-summary.html" target="_top">No Frames</a></li>
</ul>
<ul class="navList" id="allclasses_navbar_bottom">
<li><a href="../../../../../../../allclasses-noframe.html">All Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_bottom");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
</div>
<a name="skip-navbar_bottom">
<!--   -->
</a></div>
<!-- ======== END OF BOTTOM NAVBAR ======= -->
</body>
</html>
