<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<!-- NewPage -->
<html lang="en">
<head>
<!-- Generated by javadoc (version 1.7.0_17) on Tue Feb 18 09:09:01 CET 2014 -->
<title>FeatureTypeNode</title>
<meta name="date" content="2014-02-18">
<link rel="stylesheet" type="text/css" href="../../../../../../stylesheet.css" title="Style">
</head>
<body>
<script type="text/javascript"><!--
    if (location.href.indexOf('is-external=true') == -1) {
        parent.document.title="FeatureTypeNode";
    }
//-->
</script>
<noscript>
<div>JavaScript is disabled on your browser.</div>
</noscript>
<!-- ========= START OF TOP NAVBAR ======= -->
<div class="topNav"><a name="navbar_top">
<!--   -->
</a><a href="#skip-navbar_top" title="Skip navigation links"></a><a name="navbar_top_firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="../../../../../../overview-summary.html">Overview</a></li>
<li><a href="package-summary.html">Package</a></li>
<li class="navBarCell1Rev">Class</li>
<li><a href="class-use/FeatureTypeNode.html">Use</a></li>
<li><a href="package-tree.html">Tree</a></li>
<li><a href="../../../../../../deprecated-list.html">Deprecated</a></li>
<li><a href="../../../../../../index-files/index-1.html">Index</a></li>
<li><a href="../../../../../../help-doc.html">Help</a></li>
</ul>
</div>
<div class="subNav">
<ul class="navList">
<li><a href="../../../../../../com/ohnosequences/bio4j/neo4j/model/nodes/EnzymeNode.html" title="class in com.ohnosequences.bio4j.neo4j.model.nodes"><span class="strong">Prev Class</span></a></li>
<li><a href="../../../../../../com/ohnosequences/bio4j/neo4j/model/nodes/GoTermNode.html" title="class in com.ohnosequences.bio4j.neo4j.model.nodes"><span class="strong">Next Class</span></a></li>
</ul>
<ul class="navList">
<li><a href="../../../../../../index.html?com/ohnosequences/bio4j/neo4j/model/nodes/FeatureTypeNode.html" target="_top">Frames</a></li>
<li><a href="FeatureTypeNode.html" target="_top">No Frames</a></li>
</ul>
<ul class="navList" id="allclasses_navbar_top">
<li><a href="../../../../../../allclasses-noframe.html">All Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_top");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
</div>
<div>
<ul class="subNavList">
<li>Summary:&nbsp;</li>
<li>Nested&nbsp;|&nbsp;</li>
<li><a href="#field_summary">Field</a>&nbsp;|&nbsp;</li>
<li><a href="#constructor_summary">Constr</a>&nbsp;|&nbsp;</li>
<li><a href="#method_summary">Method</a></li>
</ul>
<ul class="subNavList">
<li>Detail:&nbsp;</li>
<li><a href="#field_detail">Field</a>&nbsp;|&nbsp;</li>
<li><a href="#constructor_detail">Constr</a>&nbsp;|&nbsp;</li>
<li><a href="#method_detail">Method</a></li>
</ul>
</div>
<a name="skip-navbar_top">
<!--   -->
</a></div>
<!-- ========= END OF TOP NAVBAR ========= -->
<!-- ======== START OF CLASS DATA ======== -->
<div class="header">
<div class="subTitle">com.ohnosequences.bio4j.neo4j.model.nodes</div>
<h2 title="Class FeatureTypeNode" class="title">Class FeatureTypeNode</h2>
</div>
<div class="contentContainer">
<ul class="inheritance">
<li>java.lang.Object</li>
<li>
<ul class="inheritance">
<li>com.ohnosequences.neo4j.BasicEntity</li>
<li>
<ul class="inheritance">
<li>com.ohnosequences.bio4j.neo4j.model.nodes.FeatureTypeNode</li>
</ul>
</li>
</ul>
</li>
</ul>
<div class="description">
<ul class="blockList">
<li class="blockList">
<hr>
<br>
<pre>public class <span class="strong">FeatureTypeNode</span>
extends com.ohnosequences.neo4j.BasicEntity</pre>
<div class="block">This class models precise but simple means for the annotation of sequence data.
 It describes regions or sites of interest in the protein sequence.
 In general it lists post-translational modifications, binding sites, enzyme active sites,
 local secondary structure or other characteristics reported in the cited references.
 Sequence conflicts between references are also included in this section. <br>
 Features modelled list:
 <br><br><b>Molecule processing</b><br>
 <table>
 <tr><td><b>Initiator methionine:</b></td><td>Cleavage of the initiator methionine</td></tr>
 <tr><td><b>Signal:</b></td><td>Sequence targeting proteins to the secretory pathway or periplasmic space</td></tr>
 <tr><td><b>Transit peptide:</b></td><td>Extent of a transit peptide for organelle targeting</td></tr>
 <tr><td><b>Propeptide:</b></td><td>Part of a protein that is cleaved during maturation or activation </td></tr>
 <tr><td><b>Chain:</b></td><td>Extent of a polypeptide chain in the mature protein</td></tr>
 <tr><td><b>Peptide:</b></td><td>Extent of an active peptide in the mature protein</td></tr>
 </table>
 <br><b>Regions</b><br><br>
 <table>
 <tr><td><b>Topological domain:</b></td><td> Location of non-membrane regions of membrane-spanning proteins</td></tr>
 <tr><td><b>Transmembrane:</b></td><td> Extent of a membrane-spanning region</td></tr>
 <tr><td><b>Intramembrane:</b></td><td> Extent of a region located in a membrane without crossing it</td></tr>
 <tr><td><b>Domain:</b></td><td> Position and type of each modular protein domain</td></tr>
 <tr><td><b>Repeat:</b></td><td> Positions of repeated sequence motifs or repeated domains</td></tr>
 <tr><td><b>Calcium binding:</b></td><td> Position(s) of calcium binding region(s) within the protein</td></tr>
 <tr><td><b>Zinc finger:</b></td><td> Position(s) and type(s) of zinc fingers within the protein</td></tr>
 <tr><td><b>DNA binding:</b></td><td> Position and type of a DNA-binding domain</td></tr>
 <tr><td><b>Nucleotide binding:</b></td><td> Nucleotide phosphate binding region</td></tr>
 <tr><td><b>Region:</b></td><td> Region of interest in the sequence</td></tr>
 <tr><td><b>Coiled coil:</b></td><td> Positions of regions of coiled coil whithin the protein</td></tr>
 <tr><td><b>Motif:</b></td><td> Short (up to 20 amino acids) sequence motif of biological interest</td></tr>
 <tr><td><b>Compositional bias:</b></td><td> Region of compositional bias in the protein</td></tr>
 </table>
 <br><b>Sites</b><br><br>
 <table>
 <tr><td><b>Active site:</b></td><td> Amino acid(s) directly involved in the activity of an enzyme</td></tr>
 <tr><td><b>Metal binding:</b></td><td> Binding site for a metal ion</td></tr>
 <tr><td><b>Binding site:</b></td><td> Binding site for any chemical group (co-enzyme, prosthetic group, ect.)</td></tr>
 <tr><td><b>Site:</b></td><td> Any interesting single amino acid site on the sequence</td></tr>
 </table>
 <br><b>Amino acid modifications</b><br><br>
 <table>
 <tr><td><b>Non-standard residue:</b></td><td> Occurrence of non-standard amino acids (selenocysteine an pyrrolysine) in the protein sequence</td></tr>
 <tr><td><b>Modified residue:</b></td><td> Modified residues excluding lipids, glycans and protein cross-links</td></tr>
 <tr><td><b>Lipidation:</b></td><td> Covalently attached lipid group(s)</td></tr>
 <tr><td><b>Glycosylation:</b></td><td> Covalently attached glycan group(s)</td></tr>
 <tr><td><b>Disulfide bond:</b></td><td> Cysteine residues participating in disulfide bonds</td></tr>
 <tr><td><b>Cross-link:</b></td><td> Residues participating in covalent linkage(s) between proteins</td></tr>
 </table>
 <br><b>Natural variations</b><br><br>
 <table>
 <tr><td><b>Alternative sequence:</b></td><td> Amino acid change(s) producing alternate protein isoforms</td></tr>
 <tr><td><b>Natural variant:</b></td><td> Description of a natural variant of the protein</td></tr>
 </table>
 <br><b>Experimental info</b><br><br>
 <table>
 <tr><td><b>Mutagenesis:</b></td><td> Site which has been experimentally altered by mutagenesis</td></tr>
 <tr><td><b>Sequence uncertainty:</b></td><td> Regions of uncertainty in the sequence</td></tr>
 <tr><td><b>Sequence conflict:</b></td><td> Description of sequence discrepancies of unknown origin</td></tr>
 <tr><td><b>Non-adjacent residues:</b></td><td> Indicates that two residues in a sequence are not consecutive</td></tr>
 <tr><td><b>Non-terminal residue:</b></td><td> The sequence is incomplete. Indicate that a residue is not the terminal residue of the complete protein</td></tr>
 </table>
 <br><b>Secondary structure</b><br><br>
 <table>
 <tr><td><b>Helix:</b></td><td> Helical regions within the experimentally determined protein structure</td></tr>
 <tr><td><b>Turn:</b></td><td> Turns within the experimentally determined protein structure</td></tr>
 <tr><td><b>Beta strand:</b></td><td> Beta strand regions within the experimentally determined protein structure</td></tr>
 </table></div>
<dl><dt><span class="strong">Author:</span></dt>
  <dd>Pablo Pareja Tobes <ppareja@era7.com></dd></dl>
</li>
</ul>
</div>
<div class="summary">
<ul class="blockList">
<li class="blockList">
<!-- =========== FIELD SUMMARY =========== -->
<ul class="blockList">
<li class="blockList"><a name="field_summary">
<!--   -->
</a>
<h3>Field Summary</h3>
<table class="overviewSummary" border="0" cellpadding="3" cellspacing="0" summary="Field Summary table, listing fields, and an explanation">
<caption><span>Fields</span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Field and Description</th>
</tr>
<tr class="altColor">
<td class="colFirst"><code>static java.lang.String</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/neo4j/model/nodes/FeatureTypeNode.html#FEATURE_TYPE_NAME_INDEX">FEATURE_TYPE_NAME_INDEX</a></strong></code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>static java.lang.String</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/neo4j/model/nodes/FeatureTypeNode.html#NAME_PROPERTY">NAME_PROPERTY</a></strong></code>
<div class="block">Feature type name</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>static java.lang.String</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/neo4j/model/nodes/FeatureTypeNode.html#NODE_TYPE">NODE_TYPE</a></strong></code>&nbsp;</td>
</tr>
</table>
<ul class="blockList">
<li class="blockList"><a name="fields_inherited_from_class_com.ohnosequences.neo4j.BasicEntity">
<!--   -->
</a>
<h3>Fields inherited from class&nbsp;com.ohnosequences.neo4j.BasicEntity</h3>
<code>node, NODE_TYPE_PROPERTY</code></li>
</ul>
</li>
</ul>
<!-- ======== CONSTRUCTOR SUMMARY ======== -->
<ul class="blockList">
<li class="blockList"><a name="constructor_summary">
<!--   -->
</a>
<h3>Constructor Summary</h3>
<table class="overviewSummary" border="0" cellpadding="3" cellspacing="0" summary="Constructor Summary table, listing constructors, and an explanation">
<caption><span>Constructors</span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colOne" scope="col">Constructor and Description</th>
</tr>
<tr class="altColor">
<td class="colOne"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/neo4j/model/nodes/FeatureTypeNode.html#FeatureTypeNode(org.neo4j.graphdb.Node)">FeatureTypeNode</a></strong>(org.neo4j.graphdb.Node&nbsp;n)</code>&nbsp;</td>
</tr>
</table>
</li>
</ul>
<!-- ========== METHOD SUMMARY =========== -->
<ul class="blockList">
<li class="blockList"><a name="method_summary">
<!--   -->
</a>
<h3>Method Summary</h3>
<table class="overviewSummary" border="0" cellpadding="3" cellspacing="0" summary="Method Summary table, listing methods, and an explanation">
<caption><span>Methods</span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tr class="altColor">
<td class="colFirst"><code>boolean</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/neo4j/model/nodes/FeatureTypeNode.html#equals(java.lang.Object)">equals</a></strong>(java.lang.Object&nbsp;obj)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.lang.String</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/neo4j/model/nodes/FeatureTypeNode.html#getName()">getName</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>int</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/neo4j/model/nodes/FeatureTypeNode.html#hashCode()">hashCode</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/neo4j/model/nodes/FeatureTypeNode.html#setName(java.lang.String)">setName</a></strong>(java.lang.String&nbsp;value)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.lang.String</code></td>
<td class="colLast"><code><strong><a href="../../../../../../com/ohnosequences/bio4j/neo4j/model/nodes/FeatureTypeNode.html#toString()">toString</a></strong>()</code>&nbsp;</td>
</tr>
</table>
<ul class="blockList">
<li class="blockList"><a name="methods_inherited_from_class_com.ohnosequences.neo4j.BasicEntity">
<!--   -->
</a>
<h3>Methods inherited from class&nbsp;com.ohnosequences.neo4j.BasicEntity</h3>
<code>getNode, getNodeType, setNodeType</code></li>
</ul>
<ul class="blockList">
<li class="blockList"><a name="methods_inherited_from_class_java.lang.Object">
<!--   -->
</a>
<h3>Methods inherited from class&nbsp;java.lang.Object</h3>
<code>clone, finalize, getClass, notify, notifyAll, wait, wait, wait</code></li>
</ul>
</li>
</ul>
</li>
</ul>
</div>
<div class="details">
<ul class="blockList">
<li class="blockList">
<!-- ============ FIELD DETAIL =========== -->
<ul class="blockList">
<li class="blockList"><a name="field_detail">
<!--   -->
</a>
<h3>Field Detail</h3>
<a name="FEATURE_TYPE_NAME_INDEX">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>FEATURE_TYPE_NAME_INDEX</h4>
<pre>public static final&nbsp;java.lang.String FEATURE_TYPE_NAME_INDEX</pre>
<dl><dt><span class="strong">See Also:</span></dt><dd><a href="../../../../../../constant-values.html#com.ohnosequences.bio4j.neo4j.model.nodes.FeatureTypeNode.FEATURE_TYPE_NAME_INDEX">Constant Field Values</a></dd></dl>
</li>
</ul>
<a name="NODE_TYPE">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>NODE_TYPE</h4>
<pre>public static final&nbsp;java.lang.String NODE_TYPE</pre>
</li>
</ul>
<a name="NAME_PROPERTY">
<!--   -->
</a>
<ul class="blockListLast">
<li class="blockList">
<h4>NAME_PROPERTY</h4>
<pre>public static final&nbsp;java.lang.String NAME_PROPERTY</pre>
<div class="block">Feature type name</div>
<dl><dt><span class="strong">See Also:</span></dt><dd><a href="../../../../../../constant-values.html#com.ohnosequences.bio4j.neo4j.model.nodes.FeatureTypeNode.NAME_PROPERTY">Constant Field Values</a></dd></dl>
</li>
</ul>
</li>
</ul>
<!-- ========= CONSTRUCTOR DETAIL ======== -->
<ul class="blockList">
<li class="blockList"><a name="constructor_detail">
<!--   -->
</a>
<h3>Constructor Detail</h3>
<a name="FeatureTypeNode(org.neo4j.graphdb.Node)">
<!--   -->
</a>
<ul class="blockListLast">
<li class="blockList">
<h4>FeatureTypeNode</h4>
<pre>public&nbsp;FeatureTypeNode(org.neo4j.graphdb.Node&nbsp;n)</pre>
</li>
</ul>
</li>
</ul>
<!-- ============ METHOD DETAIL ========== -->
<ul class="blockList">
<li class="blockList"><a name="method_detail">
<!--   -->
</a>
<h3>Method Detail</h3>
<a name="getName()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getName</h4>
<pre>public&nbsp;java.lang.String&nbsp;getName()</pre>
</li>
</ul>
<a name="setName(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>setName</h4>
<pre>public&nbsp;void&nbsp;setName(java.lang.String&nbsp;value)</pre>
</li>
</ul>
<a name="hashCode()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>hashCode</h4>
<pre>public&nbsp;int&nbsp;hashCode()</pre>
<dl>
<dt><strong>Overrides:</strong></dt>
<dd><code>hashCode</code>&nbsp;in class&nbsp;<code>com.ohnosequences.neo4j.BasicEntity</code></dd>
</dl>
</li>
</ul>
<a name="equals(java.lang.Object)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>equals</h4>
<pre>public&nbsp;boolean&nbsp;equals(java.lang.Object&nbsp;obj)</pre>
<dl>
<dt><strong>Overrides:</strong></dt>
<dd><code>equals</code>&nbsp;in class&nbsp;<code>com.ohnosequences.neo4j.BasicEntity</code></dd>
</dl>
</li>
</ul>
<a name="toString()">
<!--   -->
</a>
<ul class="blockListLast">
<li class="blockList">
<h4>toString</h4>
<pre>public&nbsp;java.lang.String&nbsp;toString()</pre>
<dl>
<dt><strong>Overrides:</strong></dt>
<dd><code>toString</code>&nbsp;in class&nbsp;<code>java.lang.Object</code></dd>
</dl>
</li>
</ul>
</li>
</ul>
</li>
</ul>
</div>
</div>
<!-- ========= END OF CLASS DATA ========= -->
<!-- ======= START OF BOTTOM NAVBAR ====== -->
<div class="bottomNav"><a name="navbar_bottom">
<!--   -->
</a><a href="#skip-navbar_bottom" title="Skip navigation links"></a><a name="navbar_bottom_firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="../../../../../../overview-summary.html">Overview</a></li>
<li><a href="package-summary.html">Package</a></li>
<li class="navBarCell1Rev">Class</li>
<li><a href="class-use/FeatureTypeNode.html">Use</a></li>
<li><a href="package-tree.html">Tree</a></li>
<li><a href="../../../../../../deprecated-list.html">Deprecated</a></li>
<li><a href="../../../../../../index-files/index-1.html">Index</a></li>
<li><a href="../../../../../../help-doc.html">Help</a></li>
</ul>
</div>
<div class="subNav">
<ul class="navList">
<li><a href="../../../../../../com/ohnosequences/bio4j/neo4j/model/nodes/EnzymeNode.html" title="class in com.ohnosequences.bio4j.neo4j.model.nodes"><span class="strong">Prev Class</span></a></li>
<li><a href="../../../../../../com/ohnosequences/bio4j/neo4j/model/nodes/GoTermNode.html" title="class in com.ohnosequences.bio4j.neo4j.model.nodes"><span class="strong">Next Class</span></a></li>
</ul>
<ul class="navList">
<li><a href="../../../../../../index.html?com/ohnosequences/bio4j/neo4j/model/nodes/FeatureTypeNode.html" target="_top">Frames</a></li>
<li><a href="FeatureTypeNode.html" target="_top">No Frames</a></li>
</ul>
<ul class="navList" id="allclasses_navbar_bottom">
<li><a href="../../../../../../allclasses-noframe.html">All Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_bottom");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
</div>
<div>
<ul class="subNavList">
<li>Summary:&nbsp;</li>
<li>Nested&nbsp;|&nbsp;</li>
<li><a href="#field_summary">Field</a>&nbsp;|&nbsp;</li>
<li><a href="#constructor_summary">Constr</a>&nbsp;|&nbsp;</li>
<li><a href="#method_summary">Method</a></li>
</ul>
<ul class="subNavList">
<li>Detail:&nbsp;</li>
<li><a href="#field_detail">Field</a>&nbsp;|&nbsp;</li>
<li><a href="#constructor_detail">Constr</a>&nbsp;|&nbsp;</li>
<li><a href="#method_detail">Method</a></li>
</ul>
</div>
<a name="skip-navbar_bottom">
<!--   -->
</a></div>
<!-- ======== END OF BOTTOM NAVBAR ======= -->
</body>
</html>
